examples/player/simid_player.html

```diff
@@ -35,11 +35,43 @@
       simidPlayer.initializeAd();
     }
 
+    function setButtonState(disable) {
+      let adRequests = document.querySelectorAll(".ad_request");
+      adRequests.forEach((request) => {
+        request.disabled = disable;
+      });
+    }
+
     function playAd() {
       if (!simidPlayer) {
         initAd();
       }
       simidPlayer.playAd();
+      setButtonState(/* state= */ false);
+    }
+
+    function closeAd() {
+      if (!simidPlayer) {
+        return;
+      }
+      simidPlayer.playerCloseAd();
```

Comment:
I prefer just closeAd for a name.

The rule of thumb is this. If a class or object has a name -> Then you don't need to restate its name in the API.

For example:
class Foo{
  changeFooProperty()
}

Should instead be
class Foo{
  changeProperty()
}

Then you write
foo.changeProperty()

Reply:
thank you for teaching me this